lib/commands/web-view/list-web-views.ts

```diff
@@ -0,0 +1,32 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import helpers = require("../../common/helpers");
+
+export class ListWebViewsCommand implements ICommand {
+	constructor(private $webViewService: IWebViewService,
+		private $logger: ILogger,
+		private $project: Project.IProject) { }
+	
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			this.$project.ensureCordovaProject(); 
+
+			this.$webViewService.validateFrameworkVersion();
+			let supportedWebViews = this.$webViewService.supportedWebViews;
+			
+			let platforms = _.keys(supportedWebViews);
+			let data = _.map(platforms, (platform: string, index: number) => {
+				let webViews = supportedWebViews[platform];
+				return [(++index).toString(), platform, _.map(webViews, webView => webView.name).join(",\n")];
+			});
```

Comment:
This could be simplified with a chain:

let data = _(supportedWebViews).keys().map((platform,index)=>{}).value();